### ISSUE TITLE
Chats list not focusable from keyboard

### ISSUE THREAD

**Issue by trufae** [trufae/parla#9]
• List of chat conversations on the main screen is not focusable with the keyboard, so the only way on how to enter a chat with keyboard only is by using ctrl+k chat switcher.

ref https://fedi.ml/display/421c36a5-2469-dcb4-633f-66c077973607

**Comment by pvagner** [comment]
Hello @trufae 

Huge thanks for taking care of this issue.

Unfortunately there is a follow up one in order to make the keyboard experience useful.
When chat list is focused, pressing any key moves focus to the chat input text entry.

I'd expect it to work like this:

* Open the app and make sure the "my profile" button is focused.
* Now use tab key to navigate. The focus passes through main menu button, search chats entry and chats list.
* When chat list is focused it should be possible to use arrow keys to navigate over the individual items in the list.
* It should be possible to switch into the chat corresponding with the selected item in the list by pressing the enter key. 

In fact I'm thinking that ctrl+k chat chooser shouldn't be a seperate dialog and ctrl+k should just focus search entry and typing should filter the chat list that would be my understanding so you can get right of some code duplication.

No matter how you will decide to tackle this I like it so far and thank you for working on it.
